actors/src/test/scala/zio/actors/ActorsSpec.scala

```diff
@@ -37,9 +39,10 @@ object ActorsSpec extends DefaultRunnableSpec {
             context: Context
           ): UIO[(Int, A)] =
             msg match {
-              case Reset    => UIO((0, ()))
-              case Increase => UIO((state + 1, ()))
-              case Get      => UIO((state, state))
+              case Reset               => UIO((0, ()))
+              case Increase            => UIO((state + 1, ()))
+              case Get                 => UIO((state, state))
+              case IncreaseUpTo(upper) => UIO((upper, Stream.fromIterable(state until upper)))
```

Comment:
This is a little suspicious: The state should probably evolve with the streamed response instead of directly jumping to the last value.